### DISCUSSION TITLE
Highlight/select current line focused by visual mark

### DISCUSSION THREAD

**Discussion by idkhowisyourday** [ahrm/sioyek#631] [Q&A] [answered]
I like using keys to scroll through pages however its annoying when I have to switch back to a mouse to highlight a line and use a command on it. Is there a way to highlight the current line so I can highlight/use other commands?

Essentially I am looking for a way to use sioyek but with keyboard only.

**Comment by ahrm** [comment, marked as answer]
Is `keyboard_select` not what you want?

**Reply by idkhowisyourday** [reply]
Thanks, I found it to be very useful. 

Would you a way to highlight the current line the viewer is looking at? I am not sure how useful this feature is, however there are times when I jump to a portal/highlight and want to start looking around there but when I press v the keyboard_select function is still highlighting at the original location (before jump).
